src/vs/workbench/services/search/node/ripgrepTextSearch.ts

```diff
@@ -511,7 +511,9 @@ function getRgArgs(config: IRawSearch) {
 	}
 
 	args.push('--no-config');
-	args.push('--no-ignore-global');
+	if (!config.disregardGlobalIgnoreFiles) {
+		args.push('--no-ignore-global');
```

Comment:
I think this is switched around? I know the double negative in our naming convention makes it a bit hard to reason about but if we want to disregard the global ignore files, then we should add the "no" flag... so I think the `!` needs to be removed.